node/node.go

```diff
@@ -216,17 +215,20 @@ type Node struct {
 	indexerService    *txindex.IndexerService
 	prometheusSrv     *http.Server
 
-	ipfsAPI   ipface.CoreAPI
```

Comment:
removed this for now, as we weren't using it anywhere. We can easily add it back in if needed.